src/mpriscontroller.h

```diff
@@ -25,6 +25,7 @@
 
 #include <ambermpris.h>
 #include <Mpris>
+#include <MprisMetaData>
```

Comment:
If included here, the later "class MprisMetaData" is unnecesary?

Reply:
True, I'll fix it. :slightly_smiling_face: